### ISSUE TITLE
[Feature request] Option to preserve the previous build

### ISSUE THREAD

**Issue by L0Lock** [overmindstudios/BlenderUpdater#29]
Right now the behaviour of the updater is to:
- download selected build
- extract
- copy in target
- clean up (I guess deleting the .zip file?)

Since we're dealing with daily builds, sometimes it happens that the newly downloaded build has a massive bug which makes it unusable at some point, if at all (it happened to me several times). In which case, you may want to get to your previous build if it was working. But it's not there anymore and you can't find it easily on the web.

So IMHO, it should be possible as an option to not override the previous build. Save it somewhere else.  It might be as a form of a simple checkbox in the startup screen or the builds selection screen.  
So the updater would do this:
- save the current build somewhere (the easiest way I guess would be in a `backup/` subdirectory)
- download selected build
- extract
- copy in target
- clean up (I guess deleting the .zip file?)

And in a bonus, it could be also handy to include a "backup" function in the updater. Click a backup button in the startup screen and it would copy the `backup/` content into the blender path.

**Comment by tobkum** [comment]
True! I already added that feature into BlenderUpdaterCLI, I'll see how to implement it into the GUI version as well. Thanks!

**Comment by Chrischan66** [comment]
This would really be a cool feature in the UI Version! Keep up the good work! Thank you for letting us use this tool!
